public/scripts/frames.js

```diff
@@ -198,4 +199,20 @@ function fillDataIntoLogWorkPage(id, title, estimatedTime, logWorkTotalTime, wor
     document.getElementById("logWorkTotalTime").innerHTML = logWorkTotalTime;
     document.getElementById("logWorkDone").value = workDone;
     document.getElementById("logWorkRT").innerHTML = estimatedTime - logWorkTotalTime;
+}
+
+function fillDataIntoEditPageTask(id, title, des, estimatedTime, difficulty, priority, assignee, creator, status, category, sprintNumber) {
+    console.log(title);
+    document.getElementById("editPageTaskId").innerHTML = id;
+    document.getElementById("titleEditPageTask").innerHTML = title;
+    document.getElementById("descriptionEditPageTask").value = des;
+    document.getElementById("priorityEditPageTask").value = priority;
+    document.getElementById("assigneeEditPageTask").value = assignee;
+    document.getElementById("creatorEditPageTask").innerHTML = creator;
+    document.getElementById("categoryEditPageTask").value = category;
+    document.getElementById("statusEditPageTask").value = status;
+    document.getElementById("estimatedTimeEditPageTask").value = estimatedTime;
+    document.getElementById("sprintNumberEditPageTask").value = sprintNumber;
+    document.getElementById("difficultyEditPageTask").value = difficulty;
+    //     document.getElementById("storyPointEditPageTask").innerHTML = storyPointNumber;
```

Comment:
Add story point here and also in card description.

Reply:
updated